org.eclipse.scanning.points/src/org/eclipse/scanning/points/CompoundSpgIterator.java

```diff
@@ -92,6 +94,7 @@ public PyDictionary toDict() {
 		return pyIterator.toDict();
     }
     
+	private int index = -1;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1213)